R/pandoc.R

```diff
@@ -749,7 +749,12 @@ pandoc_citeproc <- function() {
 #' @export
 pandoc_lua_filter_args <- function(lua_files) {
   # Lua filters was introduced in pandoc 2.0
-  if (pandoc2.0()) c(rbind("--lua-filter", pandoc_path_arg(lua_files)))
+  if (pandoc2.0()) {
+    if (!nzchar(Sys.getenv("RMARKDOWN_LUA_SHARED"))) {
```

Comment:
I often test the emptiness of env vars by this:

```suggestion
    if (is.na(Sys.getenv("RMARKDOWN_LUA_SHARED", NA))) {
```

It's slightly stricter, and also more readable to me ([I often have to think twice](https://yihui.org/en/2018/08/double-negatives/) what `!nzchar()` means, i.e., translate "is not non-zero chars" to "is empty string").

Reply:
You have a blog post for everything. I'll remember that. This is indeed clearer!